api/src/main/java/org/apache/iceberg/view/View.java

```diff
@@ -111,4 +112,13 @@ default ReplaceViewVersion replaceVersion() {
   default UpdateLocation updateLocation() {
     throw new UnsupportedOperationException("Updating a view's location is not supported");
   }
+
+  /**
+   * Returns the view's UUID
+   *
+   * @return the view's UUID
+   */
+  default UUID uuid() {
```

Comment:
Can it be the `String` instead of `UUID` object to be similar to the table ?

Reply:
for tables it might be that the type ends up being UUID based on https://github.com/apache/iceberg/pull/8800#discussion_r1355112214

Reply:
Yeah sorry for the delay on that PR, planning on updating to an actual UUID. @ajantha-bhat let me know what you think on that PR

Reply:
Yeah, I am ok with either String or UUID. But Expect both view and table to follow the same type. 
LGTM.

Reply:
we settled for `UUID` for Tables, so we're doing the same thing for Views